### DISCUSSION TITLE
Error: Argument of type 'S3Client' is not assignable to parameter of type

### DISCUSSION THREAD

**Discussion by hassanRsiddiqi** [civitai/civitai#590] [General]
Hi,

I recently installed Civitai on Ubuntu, using NodeJS version 19 and Ngnix as the server. However, I'm encountering some issues during the installation process when running npm run build. Here are the screenshots of the error messages I'm receiving:

Screenshot 1: ![image](https://github.com/civitai/civitai/assets/23057904/1618288f-5b8d-47f6-919b-087bf97f0dc5)

Screenshot 2: ![image](https://github.com/civitai/civitai/assets/23057904/27d5812b-90ab-4a7e-b88e-fdc38eae0ae0)

I would greatly appreciate any guidance or assistance in resolving these installation issues.

Thank you.

**Comment by hassanRsiddiqi** [comment]
@bkdiehl can you help me here?

**Reply by CustodialMatter** [reply]
@JustMaier Can you please help us?  we are hoping to set up our own mirror of civitai to publish rendered images from our telegram bot.  The idea is to introduce a "verified render" concept so people can no longer fib about the 50 hours of inpaint that went into their examples ;)

**Comment by hassanRsiddiqi** [comment]
Update, now I switched from server to local machine and trying to setup over there, but Its failing.
![Screen Shot 2023-07-05 at 11 31 01 PM](https://github.com/civitai/civitai/assets/23057904/82975f5a-1294-43b9-83e3-a9b566c5b7c8)

**Reply by Nobuchika-Matsuda** [reply]
Here is a same problem?
https://github.com/aws/aws-sdk-js-v3/issues/4451

In my case, running 'yarn cache clean --all && yarn dedupe --strategy highest' the error was banished... (but this information may be not valuable for you, sorry.)
